src/inject/dynamic-theme/image.ts

```diff
@@ -124,7 +124,7 @@ function analyzeImage(image: HTMLImageElement) {
     context.clearRect(0, 0, width, height);
 
     context.drawImage(image, 0, 0, naturalWidth, naturalHeight, 0, 0, width, height);
-    const imageData = context.getImageData(0, 0, width, height);
+    const imageData = context.getImageData(0, 0, width, height, {willReadFrequently: true} as any);
```

Comment:
You need to add `{willReadFrequently: true}` to the `getContext` function. But this is optimistic, as there aren't that many sites that will likely see a lot of images that need toe be analyzed. Not sure what browsers does with this option, but it shouldn't negatively impact other sites with just a few images.

Reply:
I was seeing this warning on almost every website, including Gmail, Linkedin...

Reply:
Might be me, but I don't visit "the major" sites from day-to-day 😅.

Reply:
Ah, you are right, it's the wrong place to put the flag 🤦